CefSharp.MinimalExample.Wpf/MainWindow.xaml

```diff
@@ -31,13 +31,13 @@
             </Grid.ColumnDefinitions>
             <Button Content="Back" Command="{Binding WebBrowser.BackCommand, ElementName=Browser}" Width="50"/>
             <Button Content="Forward" Command="{Binding WebBrowser.ForwardCommand, ElementName=Browser}" Grid.Column="1" Width="60"/>
-            <TextBox x:Name="txtBoxAddress" Text="{Binding Address, ElementName=Browser, FallbackValue=www.google.com}" Grid.Column="2" FontSize="12" BorderBrush="Gray" BorderThickness="1" />
+            <TextBox x:Name="txtBoxAddress" Text="{Binding Address, ElementName=Browser, UpdateSourceTrigger=LostFocus, Mode=TwoWay}" Grid.Column="2" FontSize="12" BorderBrush="Gray" BorderThickness="1" Keyboard.KeyDown="OnTxtBoxAddressKeyDown"/>
```

Comment:
Did you try `UpdateSourceTrigger=Explicit`?

Reply:
I did try that, but we want it to be data-bound in both ways. (updating the URL bar whenever the address changes in the browser, which breaks if we just make it be Explicit) So either we have to go with a two-way binding like this, or retain the "double bindings", but I find the latter a somewhat odd approach personally.

Reply:
A very quick check and `Explicit` looks fine on my end. Two way binding working as expected, maybe I'm missing something.

Reply:
Alright, I will give that a try; maybe I only tried that while the second binding was also in place. Will also add a "Go" button or something, since not everybody will realize that the Enter key works as expected.